chapters/mitigations-and-defensive-strategies/information-leaks/reading/README.md

```diff
@@ -240,7 +240,7 @@ Considering all of this, update the `TODO` lines of the `exploit.py` script to m
 
 Same as above, use `nm` to determine address of the `my_evil_func()` function.
 When sending your exploit to the remote server, adjust this address according to the binary running on the remote endpoint.
-The precompiled binary can be found in [the CNS public repository](/courses/cns/resources/repo "cns:resources:repo").
+The precompiled binary can be found in [the CNS public repository](https://ocw.cs.pub.ro/courses/cns/resources/repo "cns:resources:repo").
```

Comment:
Update the link, if you can, with the actual link to the precompiled binary in the [`cns` repository on GitHub](https://github.com/systems-cs-pub-ro/cns).